src/runners/handlers/slack.py

```diff
@@ -74,30 +73,46 @@ def handle(
     title = alert['TITLE']
 
     if recipient_email is not None:
-        result = sc.api_call("users.lookupByEmail", email=recipient_email)
-
-        # log.info(f'Slack user info for {email}', result)
-
-        if result['ok'] is True and 'error' not in result:
-            user = result['user']
-            userid = user['id']
-        else:
-            log.error(f'Cannot identify  Slack user for email {recipient_email}')
-            return None
-
+        if isinstance(recipient_email, str):
+                user = sc.api_call("users.lookupByEmail", email=recipient_email)
+                if not user['ok']:
+                    log.error(f'Cannot identify Slack user for email {recipient_email}')
+                    return None
+                    
+                else:
+                    userid = user['user']['id']
+                    result = sc.api_call("conversations.open", users=userid)
+                    user_id = result['channel']['id']
```

Comment:
what's the difference between an IM to one user and a channel with only one other participant?

Reply:
If only one user, we are sending it mostly as string from object construct .sql file. So, user can send email as a single string or a list(will work in case of single email also).